src/build/arg/mod.rs

```diff
@@ -492,7 +506,7 @@ impl<'help> Arg<'help> {
     /// ```
     /// [`Arg::long_help`]: ./struct.Arg.html#method.long_help
     pub fn help(mut self, h: &'help str) -> Self {
-        self.help = Some(h);
+        self.about = Some(h);
         self
     }
```

Comment:
This is where the `#[deprecated]` attribute should be placed on instead of the field.